dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+pip==6.0.8
```

Comment:
Maybe we can use >= here so it's future proof (since these dependencies are just for dev)